frontend/src/actions/parcelsActions.ts

```diff
@@ -27,15 +28,15 @@ export interface IStoreParcelsAction {
 
 export interface IStoreDraftParcelsAction {
   type: typeof ActionTypes.STORE_DRAFT_PARCEL_RESULTS;
-  draftParcelList: IProperty[];
+  draftParcelList: PointFeature[];
```

Comment:
why this change?

Reply:
With the switch to Geocoder formatted properties in the front-end, draft markers no longer worked because the were not in the correct format. This change saves draft properties in a format that is the same as the rest of our geo-formatted properties.

Reply:
so the draft markers should be appearing properly again?

Reply:
once this gets merged, yes.